### PR TITLE
test: kendra integration test

### PR DESCRIPTION
**Pull request by grinko**
*Issue #, if available:*

*Description of changes:*
Add Kendra workspace lifecycle intest test 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: integtests/clients/appsync_client.py [hunk 1]
```diff
@@ -122,6 +122,16 @@ def create_opensearch_workspace(self, input):
         )
         return self.client.execute(query).get("createOpenSearchWorkspace")
 
+    def create_kendra_workspace(self, input):
```
Comment (charles-marion) [inline integtests/clients/appsync_client.py:125]: Can you add these new functions to the following test? https://github.com/aws-samples/aws-genai-llm-chatbot/blob/main/integtests/security/unauthorized_test.py

File: integtests/chatbot-api/kendra_workspace_test.py [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+
+    s3_client = boto3.client("s3")
+    fields = result.get("fields")
+
+    # "fields" string format:
+    # {key=544aeaa9-3f3d-4995-843d-e66a993bf5e8/context.txt, AWSAccessKeyId=XXX
+    pairs = [pair.strip() for pair in fields.replace("{", "").split(',')]
+    fields_dict = dict(pair.split('=', 1) for pair in pairs)
+    bucket_name = result.get("url").split("//")[1].split(".")[0]
+    s3_client.put_object(
```
Comment (charles-marion) [inline integtests/chatbot-api/kendra_workspace_test.py:69]: Here you are using the credentials from the s3 client (current credentials). Ideally it should use the presign URL/credentials provided by this API

You should be able to pass the fields as is to a post request (without the s3 client)
https://github.com/boto/boto3/issues/1857

File: integtests/clients/appsync_client.py (hunk 1 above)
Comment (grinko) [reply integtests/clients/appsync_client.py:125]: Done

File: integtests/chatbot-api/kendra_workspace_test.py (hunk 2 above)
Comment (grinko) [reply integtests/chatbot-api/kendra_workspace_test.py]: Updated to use presigned URL and credentials provided by the API.

**Review by charles-marion — APPROVED** [review]

**Pull request merged**